### PR TITLE
Converted TimestampedEntry to a POJO.

### PR DESCRIPTION
**Pull request by pphaneuf**

### CODE REVIEW

File: java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java [hunk 1]
```diff
@@ -147,19 +148,19 @@ public static MerkleTreeLeaf parseMerkleTreeLeaf(InputStream in) {
...
    * @throws SerializationException if the data stream is too short.
    */
-  public static Ct.TimestampedEntry parseTimestampedEntry(InputStream in) {
-    Ct.TimestampedEntry.Builder timestampedEntry = Ct.TimestampedEntry.newBuilder();
+  public static TimestampedEntry parseTimestampedEntry(InputStream in) {
+    TimestampedEntry timestampedEntry = new TimestampedEntry();
 
     long timestamp = readNumber(in, CTConstants.TIMESTAMP_LENGTH);
-    timestampedEntry.setTimestamp(timestamp);
+    timestampedEntry.timestamp = timestamp;
```
Comment (phad) [inline java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java]: I know you're doing a straight conversion of the old code, but these pairs of lines look like they ought to just be folded into one;  eg. timestampedEntry.timestamp = readNumber(in, CTConstants.TIMESTAMP_LENGTH);

File: java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java [hunk 2]
```diff
@@ -109,15 +110,15 @@ public static ParsedLogEntry parseLogEntry(InputStream merkleTreeLeaf, InputStre
     MerkleTreeLeaf treeLeaf = parseMerkleTreeLeaf(merkleTreeLeaf);
     LogEntry logEntry = new LogEntry();
 
-    Ct.LogEntryType entryType = treeLeaf.timestampedEntry.getEntryType();
+    Ct.LogEntryType entryType = treeLeaf.timestampedEntry.entryType;
 
     if (entryType == Ct.LogEntryType.X509_ENTRY) {
       Ct.X509ChainEntry x509EntryChain = parseX509ChainEntry(extraData,
-        treeLeaf.timestampedEntry.getSignedEntry().getX509());
+        treeLeaf.timestampedEntry.signedEntry.getX509());
```
Comment (phad) [inline java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java]: use +4 indent for follow-on lines, here and in the next if branch

**Comment by phad** [comment]
LGTM bar minor whitespace niggle

File: java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java (hunk 2 above)
Comment (pphaneuf) [inline java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java]: Weird, I was fighting Emacs the whole time while touching those lines, as it was doing exactly that, but I figured that I didn't have the right Google Java style rules set up, so I kept putting it back the way it was... 😛

**Pull request merged**